ui/perfherder/alerts/AlertsView.jsx

```diff
@@ -202,6 +220,18 @@ class AlertsView extends React.Component {
     return params;
   };
 
+  selectNotSupportedFilters = userInput => {
```

Comment:
I think it'd be helpful to add a comment that explains why they are unsupported (reference bug number).